### PR TITLE
Fix guardian 0.14 compatibility

### PR DESCRIPTION
**Pull request by imranismail**

### CODE REVIEW

**Comment by AlexKovalevych** [comment]
@imranismail is it possible to have two different PRs. One for 0.14 compatibility, another for ecto version update. E.g. i'm using mongodb, and ecto driver is not ready for 2.1 version, so that way i can still use guardian_db with 0.14 version fix and shouldn't wait till mongodb_ecto driver have 2.1 version support?

**Comment by imranismail** [comment]
@AlexKovalevych sure, I'll separate them

**Comment by imranismail** [comment]
Okay, rewrote the history, good to go. Also someone already has a PR for Ecto update, so I'm skipping that

**Comment by doomspork** [comment]
@imranismail the ecto update has been merged in, could you please resolve the conflicts?

**Comment by imranismail** [comment]
@doomspork resolved!

File: lib/guardian_db.ex [hunk 1]
```diff
@@ -53,8 +53,14 @@ defmodule GuardianDb do
     Create a new new token based on the JWT and decoded claims
     """
     def create!(claims, jwt) do
-      prepared_claims = claims |> Dict.put("jwt", jwt) |> Dict.put("claims", claims)
-      GuardianDb.repo.insert cast(%Token{}, prepared_claims, [:jti, :typ, :aud, :iss, :sub, :exp, :jwt, :claims])
+      prepared_claims =
```
Comment (doomspork) [inline lib/guardian_db.ex:56]: Thank you for formatting this 💜

**Comment by doomspork** [comment]
Thanks again @imranismail 😀

**Pull request merged**